crates/apps_lib/src/cli.rs

```diff
@@ -3536,21 +3566,30 @@ pub mod args {
     pub const MAX_ETH_GAS: ArgOpt<u64> = arg_opt("max_eth-gas");
     pub const MEMO_OPT: ArgOpt<String> = arg_opt("memo");
     pub const MIGRATION_PATH: ArgOpt<PathBuf> = arg_opt("migration-path");
+    pub const MINIMUM_AMOUNT: ArgOpt<token::DenominatedAmount> =
+        arg_opt("minimum-amount");
     pub const MODE: ArgOpt<String> = arg_opt("mode");
     pub const NET_ADDRESS: Arg<SocketAddr> = arg("net-address");
     pub const NAMADA_START_TIME: ArgOpt<DateTimeUtc> = arg_opt("time");
     pub const NO_CONVERSIONS: ArgFlag = flag("no-conversions");
     pub const NO_EXPIRATION: ArgFlag = flag("no-expiration");
     pub const NUT: ArgFlag = flag("nut");
+    pub const OSMOSIS_REST_RPC: Arg<String> = arg("osmosis-rest-rpc");
     pub const OUT_FILE_PATH_OPT: ArgOpt<PathBuf> = arg_opt("out-file-path");
     pub const OUTPUT: ArgOpt<PathBuf> = arg_opt("output");
+    pub const OUTPUT_DENOM: Arg<String> = arg("output-denom");
     pub const OUTPUT_FOLDER_PATH: ArgOpt<PathBuf> =
         arg_opt("output-folder-path");
+    pub const OSMOSIS_POOL_HOP: ArgMulti<OsmosisPoolHop, GlobStar> =
+        arg_multi("pool-hop");
+    pub const OVERFLOW_OPT: ArgOpt<WalletAddress> = arg_opt("overflow-addr");
```

Comment:
Should we generate a disposable address on the fly like we do when paying gas via the MASP?